nifi-h2/nifi-h2-database-migrator/src/test/java/org/apache/nifi/dbmigration/h2/TestH2DatabaseUpdater.java

```diff
@@ -0,0 +1,58 @@
+package org.apache.nifi.dbmigration.h2;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+public class TestH2DatabaseUpdater {
+
+    @Test
+    public void testMigration() throws Exception {
+        final String dbPathNoExtension = "./src/test/resources/nifi-flow-audit";
+        final String dbFilePath = dbPathNoExtension + ".mv.db";
+        final File dbFile = new File(dbPathNoExtension);
```

Comment:
Does this migration test replace the original test file, or will it stay intact for subsequent test runs?